### ISSUE TITLE
Using your example Pipeline script throws a "null/" error

### ISSUE THREAD

**Issue by jonbrohauge** [Customatics/leaptest-plugin#21]
Using:
Jenkins version: 2.154
Leapworks Intergration Plugin version: 3.0.0

We try to run UI test using your example:
```
  stage ('Leapwork Integration') {
    step([$class: "LeaptestJenkinsBridgeBuilder",
      leapworkHostname: "[LeapworksServer]",
      leapworkPort: "[LeapworksServerPort]",
      leapworkAccessKey: "[LeapworksServerAccessKey]",
      leapworkDelay: "35",
      leapworkDoneStatusAs: "Success", //"Failed"
      leapworkReport: "report.xml",
      leapworkSchIds: "[LeapworksScheduleID]",
      leapworkSchNames: "[LeapworksScheduleName]"
    ]);
    sh 'sleep 25s'
    step([$class: 'JUnitResultArchiver', testResults: 'report.xml']);

    if(currentBuild.result != "FAILURE") {
      echo "RESULT: ${currentBuild.result}  SUCCESS INFO"
      // do something else
    } else {
      echo "RESULT: ${currentBuild.result}  FAIL INFO"
      // do something else
    }
  }
```
The plugin starts with showing the path to report.xml:
```
[Pipeline] step
null/report.xml
//////////////////////////////////////////////////////////////////////////////////////
Schedule [LeapworksScheduleName][LeapworksScheduleID] Launched Successfully!
----------------------------------------------------------------------------------------
RunItem: JenkinsTesterCase | Status: Passed | Elapsed: 1.5080451
2018-12-10T14:01:32.0204739+00:00 - Scheduled run executed from schedule '[LeapworksScheduleName]'

--- SNIP ---

2018-12-10T14:01:33.5361637+00:00 - Case is passed.
Environment: [LeapworksScheduleEnvironment]
Schedule: [LeapworksScheduleName]
ERROR: Failed to create a report file!
ERROR: Leapwork for Jenkins plugin finished with errors!
ERROR: java.io.IOException: No such file or directory
```

Where does the `null/` appear from?

Tried to use this as well:
```
leapworkReport: "${WORKSPACE}/report.xml",
```
The output is basically the same, save the output:
```
[Pipeline] step
null/[path/to/jenkins/job/workspace]/report.xml
//////////////////////////////////////////////////////////////////////////////////////
```

We did an `ls -lrt` and it shows a file called `report.xml` with size 0.

**Comment by romanchetto** [comment]
Hello, @jonbrohauge.
The plugin uses Jenkins Workspace variable to build full report file path:
`String junitReportPath = pluginHandler.getJunitReportFilePath(env.get(Messages.JENKINS_WORKSPACE_VARIABLE), leapworkReport);`
It looks like ${WORKSPACE} returns null. Could you please check it in your Jenkins?

**Comment by jonbrohauge** [comment]
Hello @romanchetto,

We tried with ${WORKSPACE} and it returned the path to the workspace, prefixed with `null/` as mentioned above.

**Comment by jonbrohauge** [comment]
Just tried `leapworkReport: "/tmp/target/report.xml"` and it still fails
```
[Pipeline] stage
[Pipeline] { (Leapwork Integration)
[Pipeline] step
null//tmp/target/report.xml
//////////////////////////////////////////////////////////////////////////////////////
Schedule [LeapworksScheduleName][LeapworksScheduleID] Launched Successfully!
----------------------------------------------------------------------------------------
```

**Comment by romanchetto** [comment]
@jonbrohauge Ok, I see. 
It looks like the plugin fails to get ${WORKSPACE} from build:
`EnvVars env = build.getEnvironment(listener);` 
where `listener` is `final TaskListener listener` pararmeter, passed to the plugin. 
I will try to reproduce this issue in pipeline

**Comment by jonbrohauge** [comment]
If it's any help, we are using separate build agents that are created on demand.

**Comment by romanchetto** [comment]
@jonbrohauge, the solution of this issue has already found. Tomorrow, there will be new 3.0.1 build with fix of this issue included

**Comment by romanchetto** [comment]
Hello @jonbrohauge
I have already published update 3.0.1. Please try and say, if it has helped you

**Comment by jonbrohauge** [comment]
@romanchetto Just tested v3.0.1 and it fixes out  `null/` issue.
